TheForceEngine/TFE_DarkForces/mission.cpp

```diff
@@ -443,7 +443,9 @@ namespace TFE_DarkForces
 						reticle_enable(true);
 					}
 					s_flatLighting = JFALSE;
-					s_nightvisionActive = JFALSE;
+					// Note: I am not sure why this is there but it overrides all player settings
```

Comment:
Vanilla often set things to the same setting in multiple places. If it is not needed, it can be removed.